modules/sb-provision-service-bundle.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * applications/service_brokers/provisioning-service-bundle.adoc
+
+[id='sb-provision-service-bundle-{context}']
+= Provisioning service bundles
+
+The following procedure provisions an example PostgreSQL service bundle (APB)
+that was made available by the {asb-name}.
+
+.Prerequisites
+
+* You have enabled service catalog.
+* You have installed and started the {asb-name}.
+
+.Procedure
+
+. Create a project.
+.. Navigate in the web console to *Home* -> *Projects* and click
+*Create Project*.
+.. Enter `test-postgresql-apb` in the *Name* field and click *Create*.
+
+. Create a service instance.
+.. Navigate to the *Catalog* -> *Developer Catalog* page.
+.. Select the *PostgreSQL (APB)* service bundle and click
+*Create Service Instance*.
+.. Review the default selections and set any other required fields, and click
+*Create*.
+.. Go to *Catalog* -> *Provisioned Services* and verify that the
+*dh-postgresql-apb* service instance is created and has a status of *Ready*.
++
+You can check the progress on the *Home* -> *Events* page. After a few moments,
+you should see an event for *dh-postgresql-apb* with the message "The
+instance was provisioned successfully".
+
+. Create a service binding.
++
+.. From the *Provisioned Services* page, click *dh-postgresql-apb* and click
+*Create Service Binding*.
+.. Review the default service binding name and click *Create*.
++
+This creates a new secret for binding using the name provided.
+
+. Review the secret that was created.
+.. Navigate to *Workloads* -> *Secrets* and verify that a secret was created,
```

Comment:
I think this and the next line should be reworked. Either breaking it into two sentences, or moving the secret name. It feels clunky as is. What about:

> ...verify that a secret, named **dh-postgresql-apb**, was created.